### ISSUE TITLE
process() crashes due to insufficient automation port size

### ISSUE THREAD

**Issue by atsushieno** [atsushieno/aap-lv2-sfizz#5]
(details TBD)

I need to a latest working LV2 SDK setup first but so far aap-sfizz crashes at `process()` (`run()` in sfizz LV2). It is due to insufficient atom buffer that `sfizz_lv2_send_controller()` fails to handle gracefully (it calls `lv2_atom_forge_frame_time()` which subsequently calls `lv2_atom_forge_push()` but returns non-OK, and thus skips `lv2_atom_forge_pop()` within the function, which causes inconsistent ATOM force stack.

It is for sure an inssue in sfizz but I'm not sure what is actually causing the insufficient buffer.

**Comment by atsushieno** [comment]
It turned out to be due to the minimum port size on the automation port.

```
    a lv2:OutputPort, atom:AtomPort ;
    atom:bufferType atom:Sequence ;
    atom:supports patch:Message, <http://sfztools.github.io/sfizz:OSCBlob> ;
    lv2:designation lv2:control ;
    lv2:index 1 ;
    lv2:symbol "automate" ;
    lv2:name "Automate",
      "Automatisation"@fr ;
    rsz:minimumSize 524288 ;
```

minimumSize (from LV2 [Resize Port](https://lv2plug.in/ns/ext/resize-port) extension) is 524288.

In AAP there is nothing that indicates the mininum buffer size for each port. In `androidaudioplugin-samples-host-engine`, which is also used for the default plugin compose UI, assumes that every port can be equally sized.

sfizz indicates that the *maximum* audio port frame size is 8192 samples. We cannot specify any larger number for an audio processing frame size. But as quoted above, the automation port requires minimum of 524288 bytes.

We will have to provide port sizing extension to resolve this issue.

**Comment by atsushieno** [comment]
Specifying `pp:minimumSize` in aap-sfizz `aap_metadata.xml` does not seem to automagically fix the issue yet.

**Comment by atsushieno** [comment]
aap-lv2 also has identical buffer size settings for all the channels:

https://github.com/atsushieno/aap-lv2/blob/593c48d4924c8ed3f436d04cf11035d387d3e420/androidaudioplugin-lv2/src/main/cpp/src/android-audio-plugin-lv2-bridge.cpp#L461

**Comment by atsushieno** [comment]
This particular issue should be fixed at 7fc9df4.
